gatsby-node.js

```diff
@@ -98,6 +131,7 @@ const createAuthors = async (createPage, slug) => {
 exports.createPages = async ({ actions }) => {
   const { createPage } = actions
 
+  await createSections(createPage, "news")
```

Comment:
change double quotation mark to single quotation mark. i.e. do `'news'` instead of `"news"`